src/use-query.spec.ts

```diff
@@ -1208,4 +1208,77 @@ describe('useQuery', () => {
       // No warnings!
     })
   })
+
+  describe('prefetchs query', () => {
+    it('should not prefetch a valid query', async () => {
+      const { wrapper, query } = mountSimple({
+        staleTime: 1000,
+      })
+
+      expect(wrapper.vm.data).toBeUndefined()
+
+      await flushPromises()
+
+      expect(wrapper.vm.data).toBe(42)
+
+      expect(query).toHaveBeenCalledOnce()
+
+      const queryCache = useQueryCache()
+
+      queryCache.prefetch({
+        key: ['key'],
+        query,
+      })
+
+      await flushPromises()
+
+      expect(wrapper.vm.data).toBe(42)
+
+      expect(query).toHaveBeenCalledOnce()
```

Comment:
Let's reduce a bit the empty lines
```suggestion
      expect(wrapper.vm.data).toBe(42)
      expect(query).toHaveBeenCalledOnce()
```